EventFiltering/PWGCF/CFFilterAll.cxx

```diff
@@ -89,77 +91,86 @@ enum PIDLimits { kTPCMin,
 };
 
 // For configurable tables
-static const std::vector<std::string> CFTriggerNamesALL{"ppp", "ppL", "pLL", "LLL", "ppPhi", "pd", "Ld"};
-static const std::vector<std::string> SpeciesNameAll{"Proton", "Deuteron", "Lambda"};
-static const std::vector<std::string> SpeciesName{"Proton", "Deuteron"};
-static const std::vector<std::string> SpeciesNameAnti{"AntiProton", "AntiDeuteron"};
+static const std::vector<std::string> CFTriggerNamesALL{"ppp", "ppL", "pLL", "LLL", "ppPhi", "ppRho", "pd", "Ld"};
+static const std::vector<std::string> SpeciesNameAll{"Proton", "Deuteron", "Lambda", "Pion"}; // maybe better to not include the rho here
```

Comment:
Please, for final PR delete comments.

Reply:
Changed on the request of Anton, change is reverted and some thresholds for the pions adjusted.